api/src/pcapi/routes/e2e/routes/sandbox_data.py

```diff
@@ -76,6 +76,30 @@ def setup_create_venue() -> None:
     offerers_factories.UserOffererFactory(user=pro_user, offerer=offerer)
 
 
+@app.route("/e2e/pro/update-venue", methods=["GET"])
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏cette route n'update rien ?

<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏J'ai peut être mal compris l'objectif de chaque route 🤔

Reply:
pareil, le test E2E qui appelle cette route s'appelle update venue